llvm/docs/SourceLevelDebugging.rst

```diff
@@ -169,83 +169,96 @@ Debug information descriptors are `specialized metadata nodes
 
 There are two models for defining the values of source variables at different
 states of the program and tracking these values through optimization and code
-generation: :ref:`intrinsic function calls <format_common_intrinsics>`, the
-current default, and :ref:`debug records <debug_records>`, which are a new
-non-instruction-based model
-(for an explanation of how this works and why it is desirable, see the
-`RemoveDIs <RemoveDIsDebugInfo.html>`_ document). Each module must use one or
```

Comment:
Perhaps retain some reference to the migration doc here for any archaeologists trying to understand what happened...?